### PR TITLE
feat(curriculum): add array shopping list workshop

### PR DESCRIPTION
**Pull request by jdwilkin4**
Checklist:

- [x] I have read and followed the [contribution guidelines](https://contribute.freecodecamp.org).
- [x] I have read and followed the [how to open a pull request guide](https://contribute.freecodecamp.org/how-to-open-a-pull-request/).
- [x] My pull request targets the `main` branch of freeCodeCamp.
- [x] I have tested these changes either locally on my machine, or GitPod.

### CODE REVIEW

File: curriculum/challenges/english/25-front-end-development/workshop-shopping-list/66c63bfa4685e4e3843fa0dc.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
+---
+id: 66c63bfa4685e4e3843fa0dc
+title: Step 1
+challengeType: 1
+dashedName: step-1
+---
+
+# --description--
+
+In this workshop you will continue to learn about arrays by building a grocery shopping list.
```
Comment (Ksound22) [inline curriculum/challenges/english/25-front-end-development/workshop-shopping-list/66c63bfa4685e4e3843fa0dc.md:10]:
```suggestion
In this workshop, you will continue to learn about arrays by building a grocery shopping list.
```

File: curriculum/challenges/english/25-front-end-development/workshop-shopping-list/66c730183f4020275cbf0611.md [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+title: Step 13
+challengeType: 1
+dashedName: step-13
+---
+
+# --description--
+
+For this next portion of the workshop, you will review how to remove items from the end of the array.
+
+Start by adding a `console.log` that logs the message `"This looks like to much junk food."`.
```
Comment (Ksound22) [inline curriculum/challenges/english/25-front-end-development/workshop-shopping-list/66c730183f4020275cbf0611.md:12]:
```suggestion
Start by adding a `console.log` that logs the message `"This looks like too much junk food."`.
```

File: curriculum/challenges/english/25-front-end-development/workshop-shopping-list/66c730183f4020275cbf0611.md [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+
+# --hints--
+
+You should log the message `"This looks like to much junk food."`.
```
Comment (Ksound22) [inline curriculum/challenges/english/25-front-end-development/workshop-shopping-list/66c730183f4020275cbf0611.md:16]:
```suggestion
You should log the message `"This looks like too much junk food."`.
```

Comment (Ksound22) [reply curriculum/challenges/english/25-front-end-development/workshop-shopping-list/66c730183f4020275cbf0611.md:16]: You need to change this in the rest of the seed code

**Review by Ksound22 — COMMENTED** [review]
Things look good. I only found some nitpicks.

**Review by Ksound22 — APPROVED** [review]

**Review by Sembauke — APPROVED** [review]

**Pull request merged**